### PR TITLE
fix: pin BigQuery guava version to 20

### PR DESCRIPTION
**Pull request by RyanSkraba**
**What is the current behavior?** (You can also link to an open issue here)

components-parent pins the guava version to 19.0 which causes components-bigquery to enter a spinning state that blocks builds.

**What is the new behavior?**

bigquery runtimes pin the guava version to 20.0, which is currently the version enforced by beam.

**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [X] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by RyanSkraba** [comment]
Either this or #633 could fix the currently blocking BigQuery integration tests.

File: components/components-bigquery/bigquery-integration/pom.xml [hunk 1]
```diff
@@ -21,6 +21,13 @@
     </properties>
 
     <dependencies>
+        <!-- Lock guava to the Beam dependency -->
+        <dependency>
```
Comment (iv-gonchar) [inline components/components-bigquery/bigquery-integration/pom.xml]: I suggest to move it to dependencyManagement block

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/100/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/100/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/101/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/101/)

**Pull request merged**